### ISSUE TITLE
passing options when console wallabag:install is non interactive

### ISSUE THREAD

**Issue by bricewge** [wallabag/wallabag#2159]
During the installation it is currently not possible to run `php bin/console wallabag:install --env=prod` in a non interactive way and passing it options. Like `SYMFONY_ENV=prod composer install --no-dev -o --prefer-dist` it should be possible to pass options via environment variables.

**Comment by j0k3r** [comment]
I guess you are trying to automatize the way to build your wallabag instance, like for Docker. You might find some answers on the dedicated repo: https://github.com/wallabag/docker

To answer your question, you can skip the interactive for the install command with the `-n` option (as described in the help of the command if you do `php bin/console help wallabag:install`):

```
php bin/console wallabag:install --env=prod -n
```

**Comment by bricewge** [comment]
Yes I'm trying to automate the installation with Saltstack. Someone on Gitter already mentioned my the docker repo but I didn't find what I was looking for. As you can see [here](https://github.com/wallabag/docker/blob/06af34012318268ab7f590c063a73cccf7d0d808/root/etc/ansible/entrypoint.yml#L111) there are no options passed to this command, so it will set the admin username and password to wallabag with no email.

To be more specific I want to set custom responses to [thoses](https://github.com/wallabag/wallabag/blob/master/src/Wallabag/CoreBundle/Command/InstallCommand.php#L208-L215) questions. But as far as my little understanding of the code go, it's not possible yet.

**Comment by j0k3r** [comment]
For the actual code, it's not possible.
But it's doable to give user information to the command line. But it might be a bit risky to allow giving the password in argument to the command ...

**Comment by bricewge** [comment]
Yes this why I opened this issue, it should be possible to pass those information in a correct way. Excuse me if I wasn't clear enough
